cost/budget_report_alerts/budget_report_alerts.pt

```diff
@@ -403,12 +439,9 @@ script "js_only_exceeded", type: "javascript" do
   EOS
 end
 
-escalation "esc_budget_alert" do
-  automatic true
-  label "Send Email"
-  description "Send incident email"
-  email $param_email
-end
+###############################################################################
+# Policy
+###############################################################################
 
 policy "budget_alert" do
```

Comment:
Policy block name should begin with pol_

See: https://flexera.atlassian.net/wiki/spaces/FP/pages/2184642622/Policy+Best+Practices#Block-Naming-Conventions